### ISSUE TITLE
Links are not clearly identifiable

### ISSUE THREAD

**Issue by pooriamehregan** [Informasjonsforvaltning/fdk-issue-tracker#671]
Colour alone is not enough to distinguish a link from the surrounding text.

People who are colourblind may be unable to find a link if colour is its only distinguishing feature.

Providing another visual cue -such as underlining or bolding the link text- will also make it easier for non-colourblind people (especially people with low vision) to scan the page.

Excel-ark med oversikt over linker til forekomstene:

[Issue#671 - Links are not clearly identifiable.xlsx](https://github.com/Informasjonsforvaltning/fdk-issue-tracker/files/10686757/Issue.671.-.Links.are.not.clearly.identifiable.xlsx)

[Links are not clearly identifiable - Issues - Accessibility NextGen - Accessibility.pdf](https://github.com/Informasjonsforvaltning/fdk-issue-tracker/files/10402148/Links.are.not.clearly.identifiable.-.Issues.-.Accessibility.NextGen.-.Accessibility.pdf)
